cmd/podman/shared/workers.go

```diff
@@ -0,0 +1,126 @@
+package shared
```

Comment:
Did this move from somewhere? Can we move all instances to use this and delete the old code?